### PR TITLE
Fixed issue #19094: Input Validation Vulnerability Leading to 500 on responses page

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: sanitize when upload, check when view

### CODE REVIEW

**Comment by Shnoulle** [comment]
See https://github.com/LimeSurvey/LimeSurvey/pull/3466

**Review by gabrieljenik — APPROVED** [review]
Code looks OK

File: application/helpers/expressions/em_manager_helper.php [hunk 1]
```diff
@@ -8508,6 +8508,8 @@ public static function ProcessCurrentResponses()
                                             }
                                             $aFiles[$i]->filename = $sDestinationFileName;
                                         }
+                                        /* Sanitize size */
+                                        $aFiles[$i]->size = floatval($aFiles[$i]->size);
```
Comment (gabrieljenik) [inline application/helpers/expressions/em_manager_helper.php:8512]: Use `sanitize_float` from `sanitize_helper`?

Comment (Shnoulle) [reply application/helpers/expressions/em_manager_helper.php:8512]: 1. I really don't want to allow 1,2 : it must be 1.2, real float.
2. It must be float for round, not GMP number 

https://github.com/LimeSurvey/LimeSurvey/blob/b95bf50c0dd76788d906978de4f589fd86c75ef2/application/helpers/sanitize_helper.php#L341

**Review by gabrieljenik — APPROVED** [review]
Code looks OK

**Pull request merged**